apps/main/src/dex/features/pool-list/cells/PoolTitleCell/PoolTokens.tsx

```diff
@@ -15,7 +16,7 @@ export function PoolTokens({
 }) {
   const searchedTerms = filterValue?.split(/[\s,]+/)?.map((x) => x.toLowerCase())
   return (
-    <Stack direction="row" gap={2}>
+    <Stack direction="row" gap={2} sx={responsiveTitleEllipsisSx}>
```

Comment:
Mustn't forget to use `responsiveTitleEllipsisSx` in the future when we create new title cells